frontend/src/App.js

```diff
@@ -12,25 +12,32 @@ class App extends Component {
   state = {
     sources: [],
     err: null,
+    version: {},
   }
 
   componentDidMount() {
     this.loadSources()
+    this.dbVersion()
```

Comment:
Change to something more descriptive. I'm not sure what this does by looking at this line